nin/frontend/app/scripts/directives/editor/GraphEditorInputOutput.js

```diff
@@ -77,9 +77,9 @@ class GraphEditorInputOutput extends React.Component {
         strokeWidth: 2,
         onClick: event => {this.props.editor.startOrCompleteConnection(this.props.item);},
       }),
-      this.props.scale > 1.5 ? e('text', {
+      this.props.scale >= 1.5 ? e('text', {
         x: 0,
-        y: 20,
+        y: 10 * this.props.scale,
```

Comment:
The offset is now always constant compared to the size of the node. This makes it look better at the lower zoom levels.
Font-scaling could be introduced for those who want to look really close to the nodes.